homework_21/task_1.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+
+class Employee:
+    def __init__(self, name, position, salary):
+        self.name = name
+        self.position = position
+        self.salary = int(salary)  
+
+class Department:
+    def __init__(self, name, description, employees):
+        self.name = name
+        self.description = description
+        self.employees = employees
+
+    def average(self):
+        if not self.employees:
+            return 0  #თუ თანამშრომლები არ არიან
```

Comment:
აქ შესაძლოა უკეთესი იყოს ახალი variables დამატება, მაგ: 
total_pay = sum(employee.pay for employeein self.employee)
return total_pay / len(self.employee)